supervisor/addons/addon.py

```diff
@@ -87,6 +88,12 @@
 
 WATCHDOG_TIMEOUT = aiohttp.ClientTimeout(total=10)
 
+_OPTIONS_MERGER: Final = Merger(
+    type_strategies=[(list, ["overwrite"]), (dict, ["merge"])],
+    fallback_strategies=["override"],
+    type_conflict_strategies=["override"],
+)
```

Comment:
```suggestion
_OPTIONS_MERGER: Final = Merger(
    type_strategies=[(dict, ["merge"])],
    fallback_strategies=["use_existing"],
    type_conflict_strategies=["override"],
)
```

I _think_ this should work and be safe